server/src/schema/index.ts

```diff
@@ -3,6 +3,6 @@ import { resolvers } from './resolvers';
 import { typeDefs } from './schema';
 
 export const schema = makeExecutableSchema({
-  resolvers,
+  resolvers: resolvers as any,
```

Comment:
That's because generated objects has no index signature.

```typescript
interface Resolvers {
  [key: string]: any // <-- that's missing but that's fine since it would allow to add non existing keys
}
```